eng/pipelines/source-build-sdk-diff-tests.yml

```diff
@@ -4,8 +4,8 @@ schedules:
   branches:
     include:
     - main
-    - release/*.0.1xx*
-    - internal/release/*.0.1xx*
+    - release/*.0.1xx*
+    - internal/release/*.0.1xx*

```

Comment:
What's actually the difference in these lines? Is there a whitespace change?

Reply:
I cannot find any difference between these lines.

I originally made had these changes in a draft PR to check the difference, but it showed up no matter what I did; I've updated from the release/8.0.1xx branch, compared with diff, and copied and pasted the release/8.0.1xx file. Locally, there is no difference, but the diff still appears in the PR regardless of the method I use to try and fix it.